pkg/network/common/dns.go

```diff
@@ -264,6 +281,10 @@ func (d *DNS) queryServer(nameserver, domain string) ([]net.IP, int, error) {
 }
 
 func (d *DNS) getIPsAndMinTTL(domain string) ([]net.IP, time.Duration, error) {
+	trace := utiltrace.New(fmt.Sprintf("DNS resolution for %q", domain))
+	// Probably a bit too excessive.
+	defer trace.LogIfLong(dnsQueryTraceThreshold)
```

Comment:
we shouldn't be adding it if we think it's going to be excessive...

Reply:
Actually what I meant is the threshold is probably too high and should be lower. I removed the comment.